react/features/base/media/components/native/Audio.ts

```diff
@@ -31,12 +31,25 @@ export default class Audio extends AbstractAudio {
         }
     }
 
+    /**
+     * Implements React's {@link Component#componentDidUpdate()}.
+     *
+     * @inheritdoc
+     */
+    async componentDidUpdate(prevProps: IProps): Promise<void> {
```

Comment:
I just wanted to make sure that in case something changes in the methods themselves and those that are called further, we have it covered, and it's a harmless change. WDYT?